packages/@lwc/errors/src/compiler/error-info/template-transform.ts

```diff
@@ -506,4 +506,10 @@ export const ParserDiagnostics = {
         level: DiagnosticLevel.Error,
         url: '',
     },
+    NO_DUPLICATE_SLOTS: {
+        code: 1136,
+        message: 'Slot with name "{0}" cannot be duplicated in the same template',
```

Comment:
I would love to throw a better error message here for default slots (something like `Default slot cannot be duplicated` rather than `Slot with name "" cannot be duplicated`), but I can't see any way to do that without duplicating the error itself. (Ironic?)

Reply:
I think it's possible to reuse the same error code.

```js
{
    message: "Invalid duplicate slot ({0}) in the template."
}
```

```js
const info  = name === "" ? "default" : `name="${name}"`;
return warnOnElement(ParserDiagnostics.NO_DUPLICATE_SLOTS, element.__original, [info]);
```

Reply:
Ah OK, I wasn't sure if it was okay to put English strings directly into `template-transform.ts`. I'll do this.